misc-tools/dj_make_chroot.in

```diff
@@ -241,8 +221,7 @@ if [ -e "$CHROOTDIR" ]; then
 fi
 
 mkdir -p "$CHROOTDIR"
-cd "$CHROOTDIR"
-CHROOTDIR="$PWD"
+move_into_chroot
```

Comment:
I think it's better to not put this one in a function, since it then hides the directory changed to (although it can be guessed) and also that CHROOTDIR gets reset.